packages/redux/src/products/selectors/outfits.ts

```diff
@@ -0,0 +1,63 @@
+import { getError, getIsLoading } from '../reducer/outfits';
+import { getProduct } from './product';
+import type { ProductEntity } from '../../entities/types';
+import type { ProductsState } from '../types';
+import type { StoreState } from '../../types';
+
+/**
+ * Returns the loading product outfits condition to a specific product.
+ *
+ * @param state - Application state.
+ * @param id    - Product id.
+ *
+ * @returns If the product outfits is loading or not.
+ */
+export const areProductOutfitsLoading = (
+  state: StoreState,
+  id: ProductEntity['id'],
+) => getIsLoading((state.products as ProductsState).outfits)[id];
+
+/**
+ * Returns the fetched status of a specific product outfits.
```

Comment:
```suggestion
 * Returns the fetched status of a specific product outfit.
```

Reply:
Fixed. Please review

Reply:
That's right. But I prefer your description.
Changed